src/vm/mngstdinterfaces.cpp

```diff
@@ -228,7 +228,7 @@ LPVOID MngStdItfBase::ForwardCallToManagedView(
             MethodDescCallSite mngItf(pMngItfMD, CRemotingServices::GetStubForInterfaceMethod(pMngItfMD));
 
             // Call the stub with the args we were passed originally.
-            Result = (Object*)mngItf.CallWithValueTypes_RetArgSlot(pArgs);
```

Comment:
CallWithValueTypes_RetArgSlot now takes the pointer and size of the return value because now it is the function used if the return value can be an value type larger than ARG_SLOT.  Call_RetArgSlot is exactly equivalent to the original function before my changes.